src/SpecialFunctions.jl

```diff
@@ -1,6 +1,7 @@
 module SpecialFunctions
 
 import ChainRulesCore
+import IrrationalConstants
```

Comment:
Maybe
```jl
using IrrationalConstants: invπ, sqrtπ, halfπ, logπ, twoπ, inv2π, sqrt2π, log2π, invsqrt2π
```
so we don't need `IrrationalConstants.*` in the code?

Reply:
Sure, I can change it. I did not import the constants here since it was easier to ensure that there are no unused or missing imports while working on the PR.

Reply:
I changed it.